config/bootstrap.php

```diff
@@ -93,9 +93,9 @@
  * for a very very long time, as we don't want
  * to refresh the cache while users are doing requests.
  */
-if (!Configure::read('debug')) {
-    Configure::write('Cache._cake_model_.duration', '+1 years');
-    Configure::write('Cache._cake_core_.duration', '+1 years');
+if (Configure::read('debug')) {
+    Configure::write('Cache._cake_model_.duration', '+2 minutes');
+    Configure::write('Cache._cake_core_.duration', '+2 minutes');
```

Comment:
If you aren't deploying or modifying an app for 1 year. It seems unlikely that rebuilding a cache would be that harmful, as it is liekly 'unmaintained' code at that point.